content/francais/python-turtle/the-basics.md

```diff
@@ -0,0 +1,74 @@
+
+Dans le code ci-dessus, `"orange"` est une chaîne de caractères. D'autres exemples de chaînes de caractères incluent `"123"`, `"abc!"` et `"green"`. Attention, `123`, `abc!` et `green` ne sont pas des chaînes car il manque des guillemets !
+
+## Méthodes
+
+`turtle.color()` est une **méthode.** Nous utilisons des méthodes pour que la tortue effectue certaines actions. Dans ce cas, `turtle.color()` change la couleur des lignes qu'elle dessine selon la couleur que nous lui indiquons. Nous lui avons demandé de changer la couleur en orange en lui envoyant la **chaîne** `"orange"` comme **argument**.
+
+## Arguments
+
+Les arguments sont des entrées que vous passez aux **méthodes** pour les utiliser. Dans notre exemple, `turtle.color()` a besoin d'un argument **chaîne de caractères** représentant le nom de la couleur à définir pour la tortue, donc nous avons passé `"orange"` comme **argument** à la méthode. Nous aurions pu aussi lui demander de changer la couleur pour autre chose, mais elle doit être une couleur que `turtle.color()` comprend. Les couleurs qu'elle reconnaît sont : `"red"`, `"orange"`, `"yellow"`, `"green"`, `"blue"`, `"purple"`, `"black"`, `"pink"`, et `"gray"`. Si nous envoyons autre chose que l'une de ces chaînes de couleurs, nous obtenons la couleur par défaut, qui est le noir.
+
+{{% /notice %}}
+
+Maintenant que nous avons choisi la couleur que nous voulons, nous pouvons maintenant dessiner une ligne ! Pour dessiner une ligne, il y a deux **méthodes** que nous pouvons utiliser :
+
+``` python
+turtle.forward(50)
+turtle.backward(50)
+```
+
+`turtle.forward(50)` fera avancer la tortue de 50 pixels et dessinera une ligne dans la direction à laquelle elle fait face, tandis que `turtle.backward(50)` fera la même chose, mais dans la direction opposée.
+
+{{% notice tip %}}
+
+## Pixels
+
+Une image est composée de milliers, voire de millions de pixels. Un pixel est une unité de mesure pour les images numériques, similaire à la façon dont nous mesurons le poids en livres ou les longues distances en miles.
+
+## Entiers
+
+Un entier est simplement un nombre entier, qu'il soit positif ou négatif. Dans cet exemple, `turtle.forward()` et `turtle.backward()` attendent un **nombre entier** comme argument au lieu d'une chaîne. Cela est dû au fait que l'argument représente le nombre de pixels que vous voulez que la tortue déplace en dessinant. Une chaîne de caractères ne fonctionnerait clairement pas pour ces méthodes !
+
+{{% /notice %}}
+
+Faisons avancer la tortue, donc tapez `turtle.forward(50)`. Votre code devrait ressembler à ceci :
+
+``` python
+import turtle
+
+turtle.color("orange")
+turtle.forward(50)
+```
+
+Maintenant que nous avons complété notre code, appuyez sur **run**. Vous devriez voir ce qui suit à l'écran des résultats :
+
+![texte alternatif height="600px" width="70%"](../media/basics-trinket.png "Fenêtre Trinket")
+
+Si c'est
```

Comment:
Not sure why the translation cuts off here...

Reply:
ya so weird, let me take a look and refine the prompt even better